### PR TITLE
build(deps): bumps @artsy/palette-mobile from 13.2.27 to 13.2.28

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [PHIRE-1180] 

### Description

Bumps palette-mobile and reanimated to prep work for react native `0.73.9`

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- bump reanimated and palette - gkartalis

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[PHIRE-1180]: https://artsyproduct.atlassian.net/browse/PHIRE-1180?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MrSltun — DISMISSED** [review]

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (bump reanimated and palette - gkartalis)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a4377ba5f431250fafb75a7104d496ccca2a901a
</p>

File: HACKS.md [hunk 1]
```diff
@@ -304,27 +304,6 @@ patch.
...
-
-#### When we can remove this:
-
-When we upgrade to 6.0.14 or higher, should do shortly but requires fixing a fair few type issues.
-
-### Explanation/Context
-
-React native removed removeEventListener which this library uses and causes jest tests to fail with type errors. This commit fixes the issue:
-https://github.com/react-navigation/react-navigation/commit/6e9da7304127a7c33cda2da2fa9ea1740ef56604
-
```
Comment (MrSltun) [inline HACKS.md:327]: **Question**: is this not needed anymore? 👀

File: patches/react-native-reanimated+3.3.0.patch
Comment (MrSltun) [inline patches/react-native-reanimated+3.3.0.patch:1]: 🧹🧹

**Review by MrSltun — DISMISSED** [review]
Looks great!! I just added one question, but it's non-blocking 🌟

File: HACKS.md (hunk 1 above)
Comment (gkartalis) [reply HACKS.md:327]: oh good catch I wanted to remove only the reanimated patch 🙈

**Review by MrSltun — APPROVED** [review]
`post-react-conf-updates` 🔥

**Pull request merged**